projects/python/simulation/agricultural-simulation-dataset-generator/agricultural-environment-robotti-mavicpro/controllers/mavic2pro_patrol/mavic2pro_patrol.py

```diff
@@ -0,0 +1,322 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller import Robot
+import pathlib
+import os
+import sys
+
+try:
+    import numpy as np
+except ImportError:
+    sys.exit("Warning: 'numpy' module not found.")
+
+
+def clamp(value, value_min, value_max):
+    return min(max(value, value_min), value_max)
+
+
+DATASET_NAME = 'dataset_location/UAV'
```

Comment:
It might be good to gather all parameters in one file and import them from there.

Reply:
Okay, will collect the parameters in one header file.